multicluster/test/e2e/service_test.go

```diff
@@ -89,24 +99,48 @@ func (data *TestData) probeMCServiceFromCluster(t *testing.T, clusterName string
 	return ip
 }
 
-func (data *TestData) testServiceExport(t *testing.T) {
-	clientPodName := "test-service-client"
-	data.setupTestResources(t)
-	defer data.tearDownTestResources(t)
-
+// Deploy service exports in east and west clusters
+func setUpServiceExport(data *TestData, t *testing.T) {
 	if err := data.deployServiceExport(westCluster); err != nil {
 		t.Fatalf("Error when deploy ServiceExport in west cluster: %v", err)
 	}
-	defer data.deleteServiceExport(westCluster)
+	//defer data.deleteServiceExport(westCluster)
```

Comment:
maybe just simply remove it?

Reply:
Yes. Just keeping this line as this is WIP. Will remove. Thanks!